testing/commontest.py

```diff
@@ -140,7 +140,10 @@ def _get_locations(src_local, dest_local, src_dir, dest_dir):
     """
     Return a tuple of remote or local source and destination locations
     """
-    remote_location = "cd {rdir}; {tdir}/server.py::{dir}"
+    if os.name == "nt":
+        remote_location = "cd {rdir} & {tdir}\\server.py::{dir}"
```

Comment:
@ericzolf I don't understand why we need double backslash `\\`

I would expect a single backslash.

Reply:
Backslash is strings or bytes is an escape character,  so you need to escape it with itself. Unless you use raw, the r-prefix, _but_ you still might need to escape it if you use the string as regex.